### PR TITLE
feat: [SRM-10838]: Health source cypress fix

### PR DESCRIPTION
**Pull request by deepesh-ui**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by praneshg239 — APPROVED** [review]

**Review by vasuki-kalluri — APPROVED** [review]

File: cypress/support/commands.ts [hunk 1]
```diff
@@ -176,6 +176,7 @@ Cypress.Commands.add('visitVerifyStepInPipeline', () => {
 
 // Change Intelligence commands
 Cypress.Commands.add('visitChangeIntelligence', () => {
+  cy.visitPageAssertion('[class^=SideNav-module_main]')
```
Comment (harshilbits) [inline cypress/support/commands.ts:179]: @deepesh-ui why do we need this buddy ?

Comment (deepesh-ui) [reply cypress/support/commands.ts:179]: Fix: When cypress runs the test first time, page loads takes a while leading to timeout of element not found. Adding cy.visitPageAssertion() to assert page load before calling API.

Comment (deepesh-ui) [reply cypress/support/commands.ts:179]: Old PR for reference : https://github.com/harness/harness-core-ui/pull/8788

Comment (harshilbits) [reply cypress/support/commands.ts:179]: okay why are we passing any argument here ? Cant see any argument this function takes in the referenced PR.#8788

**Review by harshilbits — APPROVED** [review]

**Pull request merged**